install_files/ansible-base/roles/app/tasks/initialize_securedrop_app.yml

```diff
@@ -141,6 +141,15 @@
     - gpg
     - securedrop_config
 
+- name: Copy custom notification text to config.py.
+  lineinfile:
+     dest: "{{ securedrop_code }}/config.py"
+     regexp: "^CUSTOM_NOTIFICATION"
+     line: "CUSTOM_NOTIFICATION = '{{ custom_notification_text }}'"
+  when: custom_notification_text is defined and custom_notification_text != ''
```

Comment:
This overwrites `config.py` if you change `prod-specific.yml` and reprovision. I don't consider this because I edit `config.py` myself to test things, but I suppose our intention is that admins never touch it themselves (is that correct @conorsch?). Maybe we should set the `group_var` for the staging and development VMs to automatically note that this is a testing instance, and specifically, whether it's a staging or development box as a developer convenience, especially in the context of https://github.com/freedomofpress/securedrop/pull/1458.